src/display/canvas.js

```diff
@@ -1280,9 +1282,58 @@ class CanvasGraphics {
     // transparent canvas when we have blend modes.
     const width = this.ctx.canvas.width;
     const height = this.ctx.canvas.height;
-
+    this.defaultBackgroundColor = background || "#ffffff";
```

Comment:
Can't this just be a normal `const defaultBackgroundColor = ...` variable instead, since it doesn't appear that you're referencing it anywhere else in the class?